mission_control/navigator_missions/nav_missions/pinger.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
+from __future__ import division
+import txros
+import std_srvs.srv
+import numpy as np
+import tf
+import tf.transformations as trns
+from navigator_msgs.msg import ShooterDoAction, ShooterDoActionGoal
+from navigator_msgs.srv import CameraToLidarTransform, CameraToLidarTransformRequest, FindPinger, FindPingerRequest, SetFrequency, SetFrequencyRequest
+from geometry_msgs.msg import Point
+from std_srvs.srv import SetBool, SetBoolRequest
+from twisted.internet import defer
+from image_geometry import PinholeCameraModel
+from visualization_msgs.msg import Marker,MarkerArray
+import navigator_tools
+
+class PingerMission:
```

Comment:
One issue in the future is that there may be multiple acoustic pinger challenges on the course at a time (with different frequencies). There needs to be a way for the mission planner to tell this mission the id's of the gates it is investigating, and the frequency of the pinger to look for.